sidebars.js

```diff
@@ -851,6 +852,841 @@ const sidebars = {
+                  label: 'ベンチマークを実行する',
+                },
+              ],
+            },
+          ],
+        },
+        {
+          type: 'category',
+          label: '分析クエリを実行する',
+          collapsible: true,
+          items: [
+            {
+              type: 'doc',
+              id: 'scalardb-samples/scalardb-analytics-postgresql-sample/README',
+              label: 'ScalarDB Analytics with PostgreSQL を実行する',
+            },
+            // This doc no longer exists but we should have some type of doc here.
+            // {
+            //   type: 'doc',
+            //   id: 'scalardb-analytics-spark/getting-started',
+            //   label: 'Run ScalarDB Analytics',
+            // },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'Run Non-Transactional Storage Operations',
+          collapsible: true,
+          items: [
+            {
+              type: 'category',
+              label: '非トランザクションストレージ操作を実行する',
+              collapsible: true,
+              items: [
+                {
+                  type: 'doc',
+                  id: 'run-non-transactional-storage-operations-through-library',
+                  label: 'ScalarDB Core ライブラリーを使用する',
+                },
+                {
+                  type: 'doc',
+                  id: 'scalardb-cluster/run-non-transactional-storage-operations-through-scalardb-cluster',
+                  label: 'ScalarDB Cluster を使用する',
+                },
+              ],
+            },
+            {
+              type: 'doc',
+              id: 'scalardb-cluster/run-non-transactional-storage-operations-through-sql-interface',
+              label: 'SQL インターフェースを使用して実行する',
+            },
+            {
+              type: 'doc',
+              id: 'run-non-transactional-storage-operations-through-primitive-crud-interface',
+              label: 'プリミティブ CRUD インターフェースを使用して実行する',
+            },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'サンプルアプリケーションを実行する',
+          collapsible: true,
+          items: [
+            {
+              type: 'doc',
+              id: 'scalardb-samples/multi-storage-transaction-sample/README',
+              label: 'マルチストレージトランザクション',
+            },
+            {
+              type: 'doc',
+              id: 'scalardb-samples/microservice-transaction-sample/README',
+              label: 'マイクロサービストランザクション',
+            },
+            {
+              type: 'doc',
+              id: 'scalardb-samples/spring-data-multi-storage-transaction-sample/README',
+              label: 'Spring Data JDBC によるマルチストレージトランザクション',
+            },
+            {
+              type: 'doc',
+              id: 'scalardb-samples/spring-data-microservice-transaction-sample/README',
+              label: 'Spring Data JDBC によるマイクロサービストランザクション',
+            },
+          ],
+        },
+        {
+          type: 'category',
+          label: '管理',
```

Comment:
```suggestion
          label: '詳細',
```

Same as above. In this case, the contents in this category are not minor. So, I think `詳細` is good.